dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertSenderService.java

```diff
@@ -92,6 +99,19 @@ public void send(List<Alert> alerts) {
                 alertDao.updateAlert(AlertStatus.EXECUTION_FAILURE, "no bind plugin instance", alertId);
                 continue;
             }
+
+            if (StringUtils.isNotEmpty(alert.getContent())) {
```

Comment:
I found the issue is just about email, but if change the content here, will take affect to all alert type, it's better to put before the email handle content, WDYT?

Reply:
BTW, had you found which alert type and content will cause this issue?

Reply:
> I found the issue is just about email, but if change the content here, will take affect to all alert type, it's better to put before the email handle content, WDYT?

Thank you for your advice

Reply:
> 

I find that conversion issues occur when the alert type is email and the content is an object

Reply:
> @jackfanwan BTW, had you found which alert type and content will cause this issue?

I find that conversion issues occur when the alert type is email and the content is an object